[TA] Add analyze tasks feature support (#17267)
Azure/azure-sdk-for-java: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

# Release History
## 5.1.0-beta.3 (Unreleased)
**New features**

> - Added support for healthcare analysis feature, it is a long-running operation, and the cancellation supported. 
> - Added support for analyze tasks feature, It analyzes multiple tasks, such as, entity recognition, PII entity recognition 
> and key phrases extraction simultaneously in a list of document.

Reviewer: ```suggestion
- Added support for healthcare analysis feature. Tt is represented as a long-running operation. Cancellation supported. 
- Added support for analyze multiple tasks (such as, entity recognition, PII entity recognition 
and key phrases extraction) simultaneously in a list of document.
```
maybe?
